source/reference/permissions.txt

```diff
@@ -0,0 +1,32 @@
+.. _c2c-permissions-and-roles:
+
+================
+User Permissions
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/fact-permissions-body.rst
+
+atlas/reference/user-roles/
```

Comment:
**Typo**

Line 17: Looks like this was left on the page on accident.

Reply:
Done